Sources/Algebra/AdditiveGroup.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  AdditiveGroup.swift
+//  Algebra
+//
+//  Created by Benjamin Wetherfield on 10/11/2018.
+//
+
+public protocol AdditiveGroup: Additive, Invertible {
+    
+    static prefix func - (_ element: Self) -> Self
```

Comment:
Before we merge this in, can you add doc comments to anything publicly available?